vardbg/assets/styles/default.py

```diff
@@ -0,0 +1,47 @@
+from pygments.style import Style
+from pygments.token import (
+    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
+)
+from vardbg.output.video_writer.getstyle import get_style_by_name
+
+scheme = get_style_by_name('wood')
```

Comment:
There is no need for a separate "default" file. That can be handled as a simple alias in `get_style_by_name`.